.github/workflows/openai.yml

```diff
@@ -5,20 +5,20 @@ name: OpenAI
 
 on:
   pull_request_target:
```

Comment:
To run this new workflow we'll need to change to `pull_request` temporarily.

Reply:
Okay we will change it back once the openai tests pass